lib/image_properties.dart

```diff
@@ -30,5 +31,6 @@ class ImageProperties {
     this.centerSlice,
     this.matchTextDirection = false,
     this.filterQuality = FilterQuality.low,
+    this.prefixNetworkImageRelativePath = ''
```

Comment:
Can you add an `assert` to the constructor that ensures that this is never null?

Reply:
constructor `ImageProperties` is `const`, so it cannot have a body.